index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./css/style.css">
+    <title>Youtube Replica</title>
+</head>
+<body>
+    <div class="nav"><!--flex container-->
+        <div class="logo items"><img src="./images/YouTube_Logo.png" alt="logo" class ='logo'> </div>
+        
+        <div class="search-container items"><!--flex container-->
+            <div><input class="items-search" type="text" placeholder="Search"></div>
+            <div><input type="button" value= "Search"name="Search" id=""></div>
+        </div>
+       
+        <div class="items">
+            <!-- other icons here -->
+        </div>
+        
+    
+    </div>
+
+    <div class ='content'><!--flex container-->
+        <div class="video content-items"><!--flex container-->
+            <div>
+                <iframe width="560" height="315" src="https://www.youtube.com/embed/FnBThWCy6u0" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+            </div>
+            
```

Comment:
I think the <div lass ='content'> should be <section lass ='content'> and segment group of related task in sections